castervoice/rules/ccr/rust_rules/rust.py

```diff
@@ -63,9 +63,9 @@ class Rust(MergeRule):
             R(Text("struct ")),
         #
         SymbolSpecs.COMMENT:
-            R(Text("//")),
+            R(Text("// ")),
         SymbolSpecs.LONG_COMMENT:
-            R(Text("///")),
+            R(Text("/// ")),
```

Comment:
This one might be subjective, but I think most people always have a space after the comments.